connector/connect/common/src/main/protobuf/spark/connect/commands.proto

```diff
@@ -215,15 +215,24 @@ message WriteStreamOperationStart {
   }
 
   StreamingForeachWriter foreach_writer = 13;
+  StreamingForeachFunction foreach_batch = 14;
```

Comment:
I think you can rename the protos if it has not yet be released.

Reply:
Nice. Will change in this.

Reply:
Replaced.